mp/src/game/shared/momentum/mom_system_gamemode.cpp

```diff
@@ -122,9 +122,13 @@ void CGameMode_KZ::SetGameModeVars()
 {
     CGameModeBase::SetGameModeVars();
 
-    // KZ-specific
+    // KZ-specific (Based on SimpleKZ variables)
+    sv_accelerate.SetValue("6.5");
     sv_airaccelerate.SetValue(100);
-    sv_maxspeed.SetValue(250);
+    sv_friction.SetValue("5.2");
```

Comment:
same here
```suggestion
    sv_friction.SetValue(5.2f);
```

Reply:
thanks didn't know that, will change it in the next iteration.